packages/pluggableWidgets/datagrid-web/src/components/Header.tsx

```diff
@@ -65,7 +66,9 @@ export function Header<D extends object>({
                     {column.render("Header")}
                     {sortIcon && <FontAwesomeIcon icon={sortIcon} />}
                 </div>
-                {filterable && column.canFilter && <div className="filter">{column.render("Filter")}</div>}
+                {filterable && column.canFilter && (
+                    <div className="filter">{column.customFilter ? column.customFilter : column.render("Filter")}</div>
```

Comment:
- 🎄 use `??`?
- I think we break wysiwyg again: `<div className="filter">` should be inside `                        <column.customFilter.renderer>` for editor preview. Maybe we can use the same trick with `renderWrapper` as in `cellRenderer` to achieve it.

Reply:
Good one.. Done!